hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -135,6 +136,9 @@ def get_match_details(table: Table, content_id: str) -> t.List[MatchDetail]:
             signal_source=record.signal_source,
             signal_type=record.signal_type.get_name(),
             updated_at=record.updated_at.isoformat(),
+            match_distance=int(record.match_distance)
```

Comment:
Is this necessary. Does DDB return the match_distance in some other format? Decimal etc?

Reply:
yes but only our API middleware seemed to care. if passed along directly it gets turned into Decimal(<val>)